config/go.d/prometheus.conf

```diff
@@ -1546,3 +1546,5 @@ jobs:
   - name: besu_local
     url: '127.0.0.1:9545'
     expected_prefix: 'besu_'
+  - name: crowdsec_local
+    url: http://127.0.0.1:6060/metrics
```

Comment:
Can you please add `expected_prefix`?

Reply:
Is there a syntax for multiple prefixes?

Reply:
@luisj1983 we need just one that would be unique for crowdsec. It is ok if this exporter has metrics with other prefixes.

Reply:
@ilyam8 OK, done. Anything else I need to do prior to a merge?

Reply:
No, thanks